Source/Scene/PostProcessStage.js

```diff
@@ -0,0 +1,698 @@
+        createSampler(this);
+
+        if (!this._ready) {
+            return;
+        }
+
+        var framebuffer = this._textureCache.getFramebuffer(this._name);
+        this._command.framebuffer = framebuffer;
+
+        if (!defined(framebuffer)) {
+            return;
+        }
+
+        var colorTexture = framebuffer.getColorTexture(0);
+        var renderState = this._renderState;
+        if (!defined(renderState) || colorTexture.width !== renderState.viewport.width || colorTexture.height !== renderState.viewport.height) {
+            this._renderState = RenderState.fromCache({
+                viewport : new BoundingRectangle(0, 0, colorTexture.width, colorTexture.height)
+            });
+        }
+
+        this._command.renderState = renderState;
+    };
+
+    /**
+     * Executes the post-process stage. The color texture is the texture rendered to by the scene or from the previous stage.
+     * @param {Context} context The context.
+     * @param {Texture} colorTexture The input color texture.
+     * @param {Texture} depthTexture The input depth texture.
+     * @private
+     */
+    PostProcessStage.prototype.execute = function(context, colorTexture, depthTexture) {
+        if (!defined(this._command) || !defined(this._command.framebuffer) || !this._ready || !this._enabled) {
+            return;
+        }
+
+        this._colorTexture = colorTexture;
+        this._depthTexture = depthTexture;
+
+        if (!Sampler.equals(this._colorTexture.sampler, this._sampler)) {
+            this._colorTexture.sampler = this._sampler;
+        }
+
+        var passState = this.scissorRectangle.width > 0 && this.scissorRectangle.height > 0 ? this._passState : undefined;
+        if (defined(passState)) {
+            passState.context = context;
+        }
+
+        this._command.execute(context, passState);
+    };
+
+    /**
+     * Returns true if this object was destroyed; otherwise, false.
+     * <p>
+     * If this object was destroyed, it should not be used; calling any function other than
+     * <code>isDestroyed</code> will result in a {@link DeveloperError} exception.
+     * </p>
+     *
+     * @returns {Boolean} <code>true</code> if this object was destroyed; otherwise, <code>false</code>.
+     *
+     * @see PostProcessStage#destroy
+     */
+    PostProcessStage.prototype.isDestroyed = function() {
+        return false;
+    };
+
+    /**
+     * Destroys the WebGL resources held by this object.  Destroying an object allows for deterministic
+     * release of WebGL resources, instead of relying on the garbage collector to destroy this object.
+     * <p>
+     * Once an object is destroyed, it should not be used; calling any function other than
+     * <code>isDestroyed</code> will result in a {@link DeveloperError} exception.  Therefore,
+     * assign the return value (<code>undefined</code>) to the object as done in the example.
+     * </p>
+     *
+     * @returns {undefined}
```

Comment:
For all the new files remove `@returns {undefined}` which were removed in https://github.com/AnalyticalGraphicsInc/cesium/pull/6326.